src/DiffEqFlux.jl

```diff
@@ -1,23 +1,13 @@
 module DiffEqFlux
 
 using Adapt, Base.Iterators, ConsoleProgressMonitor, DataInterpolations,
-    DiffEqBase, SciMLSensitivity, DiffResults, Distributions, DistributionsAD,
-    ForwardDiff, Optimization, OptimizationPolyalgorithms, LinearAlgebra,
+    DiffEqBase, Distributions, DistributionsAD,
+    ForwardDiff, LinearAlgebra, Lux,
     Logging, LoggingExtras, Printf, ProgressLogging, Random, RecursiveArrayTools,
-    Reexport, SciMLBase, StaticArrays, TerminalLoggers, Zygote, ZygoteRules
+    Reexport, SciMLBase, TerminalLoggers, Zygote, ZygoteRules
 
 @reexport using SciMLSensitivity
-@reexport using Zygote
-
-# deprecate
-
-import OptimizationFlux
-import NNlib
-import Lux
-using Requires
-using Cassette
 @reexport using Flux
```

Comment:
Now that we support `Lux`, reexporting `Flux` could make qualification errors in codes that only have `Lux`.
Error like:
```julia
WARNING: both Lux and Flux export "Chain"; uses of it in module Main must be qualified
```

Reply:
#795

Reply:
Yeah we should remove the reexport. Add that to the list of things.